### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by magicansk**
#### Pre-Submission Checklist

- [ ] Your pull request targets the `staging` branch of freeCodeCamp.
- [ ] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [ ] You have only one commit (if not, [squash](http://forum.freecodecamp.org/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [ ] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [* ] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [ ] Addressed currently open issue (replace XXXXX with an issue no in next line)

Closes #XXXXX

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -89,7 +89,10 @@ This will download the entire FCC repo to your projects directory.
 
 #### Setup Your Upstream
 
-1. Change directory to the new freeCodeCamp directory (`cd freeCodeCamp`)
+1. Change directory to the new freeCodeCamp directory  
+```bash
```
Comment (raisedadead) [inline CONTRIBUTING.md]: This should be `shell` instead same as in rest of the guide.

**Review by raisedadead — CHANGES_REQUESTED** [review]

**Comment by raisedadead** [comment]
Please refer my https://github.com/freeCodeCamp/freeCodeCamp/pull/16013#pullrequestreview-71282727 and do the same here.

File: CONTRIBUTING.md (hunk 1 above)
Comment (magicansk) [reply CONTRIBUTING.md]: hey, `bash` is correct and take effect!

**Comment by magicansk** [comment]
hey, `bash` is correct and take effect!

**Comment by magicansk** [comment]
Already PR, and fixed it.

**Pull request closed without merging**